src/base-config/keyboard.json

```diff
@@ -31,6 +31,9 @@
     "file.previewHighlight":  [
         "Ctrl-Shift-C"
     ],
+    "file.previewWordNavigation":  [
```

Comment:
We usually add shortcuts only if it is very frequently used. since people are not likely to toggle this feature, a shortcut isn't necessary.